src/inmanta/protocol/methods.py

```diff
@@ -818,9 +845,9 @@ def get_reports(tid: uuid.UUID, start: str = None, end: str = None, limit: int =
     Return compile reports newer then start
 
     :param tid: The id of the environment to get a report from
-    :param start: Reports after start
-    :param end: Reports before end
-    :param limit: Maximum number of results, up to a maximum of 1000
+    :param start: Optional. Reports after start
+    :param end: Optional. Reports before end
+    :param limit: Optional. Maximum number of results, up to a maximum of 1000
```

Comment:
We should mention the default behavior when the limit is None. The same of the other endpoints with this limit parameter.